projects/fluxcd/flux2/Makefile

```diff
@@ -13,8 +17,21 @@ FLUX_MANIFESTS_TARGET=$(REPO)/cmd/flux/manifests
 
 BINARY_PLATFORMS=linux/amd64 linux/arm64 darwin/amd64 darwin/arm64
 
+BUILD_TARGETS=local-images
+RELEASE_TARGETS=images
+
 HAS_S3_ARTIFACTS=true
-IMAGE_NAMES=
+HAS_HELM_CHART=true
+HELM_DESTINATION_REPOSITORY=fluxcd/flux2
+HELM_SOURCE_OWNER=fluxcd-community
+HELM_SOURCE_REPOSITORY=helm-charts
+HELM_GIT_TAG?=$(shell cat HELM_GIT_TAG)
+HELM_DIRECTORY=charts/flux2
+HELM_ADDITIONAL_KEY_VALUES=\
+  HELM_CONTROLLER_TAG=$(shell cat ../helm-controller/GIT_TAG)-$(GIT_HASH) \
```

Comment:
I suspect this is going to be problematic... GIT_HASH is the current commit hash for this given build which may nor not build all these projects.  You may also want to go ahead and brainstorm with @abhay-krishna how to handle this for staging/prod images since the final image tags are not in this format and not known until release time.  Are we planning on applying values.yaml during deployment and these "hardcoded" values are really just fallback?

Reply:
I don't have a good resolution on this

Reply:
Yeah I was thinking about this more to. Let's see if we can get on a chime and talk it out later.